pkg/kubelet/prober/prober.go

```diff
@@ -99,6 +94,21 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 		return results.Success, nil
 	}
 
+	// lazy initialize probes
```

Comment:
Are we confident there's no concurrent access to this type?  Maybe comment that?

Reply:
yes, there are with the tcp dialer, let me bring this back, it seems I was over optimizing

Reply:
I see, I wasn't understanding this correctly, all the workers share the same prober.
This will require to change that so each worker has its own probe, and not share the manager global probe